text_extensions_for_pandas/io/conll.py

```diff
@@ -343,7 +352,8 @@ def _iob_to_iob2(df: pd.DataFrame, column_names: List[str],
 def _doc_to_df(doc: List[_SentenceData],
                column_names: List[str],
                iob_columns: List[bool],
-               space_before_punct: bool) -> pd.DataFrame:
+               space_before_punct: bool,
+               doc_line_nums: List[int]) -> pd.DataFrame:
```

Comment:
This additional argument can go away if line numbers are included in the `_SentenceData` data structure.